dev/PushNotifications/externs.h

```diff
@@ -28,5 +35,13 @@ inline winrt::Windows::Foundation::IInspectable GetArgsFromComStore()
     THROW_HR_IF(HRESULT_FROM_WIN32(ERROR_TIMEOUT), !GetWaitHandleForArgs().wait(receiveArgsTimeoutInMSec));
 
     // If COM static store was uninit, let it throw
-    return winrt::Windows::ApplicationModel::Core::CoreApplication::Properties().Lookup(ACTIVATED_EVENT_ARGS_KEY);
+    if (PushNotificationHelpers::IsPackagedAppScenario())
```

Comment:
If (IsPackagedApp())
{
  use the COM activation key;
}
else
{
 use the protocol activator key for Push
}

Reply:
I believe the scenario is inconsequential now.